app/uk/gov/hmrc/tradergoodsprofiles/connectors/RouterConnector.scala

```diff
@@ -87,13 +87,12 @@ class RouterConnector @Inject() (
 
   def removeRecord(eori: String, recordId: String, actorId: String)(implicit hc: HeaderCarrier): Future[HttpResponse] =
     withMetricsTimerAsync("tgp.removerecord.connector") { _ =>
-      val url = appConfig.routerUrl.withPath(routerRoute(eori, recordId))
+      val url = appConfig.routerUrl.toUrl + routerRouteRemoveRecord(eori, recordId, actorId)
```

Comment:
do not use the + sign to concatenate a string as this give a compiler warning.

Reply:
I'm aware of that. The issue is when you try to parse a string with a query parameter using `UrlPath.parse("string?queryParam=value")`, you get this error:

```
io.lemonlabs.uri.parsing.UriParsingException: Invalid Path could not be parsed. Error(57, NonEmptyList(EndOfString(57,73)))
```

That's why I followed the same approach as `getRecords()`

Any alternative is welcomed! ✌️

Reply:
there are different way, we can talk after the SU

Reply:
one is:
 private def routerRouteRemoveRecord(eoriNumber: String, recordId: String, actorId: String) =
    appConfig.routerUrl
      .withPath(UrlPath.parse(s"$routerBaseRoute/$eoriNumber/records/$recordId"))
      .withQueryString(QueryString.fromPairs("actorId" -> actorId))